Mathlib/Analysis/Normed/Operator/BoundedLinearMaps.lean

```diff
@@ -466,6 +466,22 @@ theorem ContinuousOn.clm_apply {X} [TopologicalSpace X] {f : X → (E →L[𝕜]
     ContinuousOn (fun x ↦ f x (g x)) s :=
   isBoundedBilinearMap_apply.continuous.comp_continuousOn (hf.prodMk hg)
 
+theorem ContinuousOn.continuousLinearMapCoprod {X : Type*} [TopologicalSpace X]
```

Comment:
Local style in that file is to write `{X}`: would you like to switch to that?
```suggestion
theorem ContinuousOn.continuousLinearMapCoprod {X} [TopologicalSpace X]

Reply:
That's definitely a step backward, as it introduces universe metavariables

Reply:
Thanks! Let's wait until the other PR is merged, then I'll maintainer merge this PR also. (Unless Yael beats me to it.)